File: UserProfileService/UserProfileService.Application/DTOs/ProfileDTOs/UpdateProfileDTO.cs
using UserProfileService.Domain.Entities;
using UserProfileService.Domain.Enums;

namespace UserProfileService.Application.DTOs
{
    public class UpdateProfileDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Weight { get; set; }
        public double Height { get; set; }
        public ActivityLevel ActivityLevel { get; set; }
        public int DesiredGlassesOfWater { get; set; }
    }
}

File: NutritionBalance.Client/src/app/models/profile/DTOs/profile/update-profile.dto.ts
import { ActivityLevel } from '../../enums/activity-level.enum';

export class UpdateProfileDTO {
  id!: number;
  name!: string;
  weight!: number;
  height!: number;
  activityLevel!: ActivityLevel;
  desiredGlassesOfWater!: number;
}
